### DISCUSSION TITLE
How to download Full HD (1080p) video has audio?

### DISCUSSION THREAD

**Discussion by Viet27th** [fent/node-ytdl-core#1118] [General]
The photo below is all the videos and audio I got. However, 1080 video has no audio. Is there a way to download it with the audio?

![image](https://user-images.githubusercontent.com/30460839/185749872-7a0f626a-72ae-4254-be14-20e8316617d8.png)

There is a browser called Coccoc that allows downloading videos in Full HD quality along with audio. I also want my app to do the same. Thanks everyone!

**Comment by Secreto31126** [comment]
[From the README](https://github.com/fent/node-ytdl-core#handling-separate-streams)

**Reply by Viet27th** [reply]
I will try it out. Thank you so much!
